### PR TITLE
Beamer Integration

### PR DESCRIPTION
**Pull request by inderjeet-developer7**

### CODE REVIEW

File: app/lib/main.dart [hunk 1]
```diff
@@ -55,19 +53,23 @@ Future<void> startApp() async {
...
+    locationBuilder: Routes.getRoutes(),
+  );
 
   @override
   Widget build(BuildContext context) {
     debugInvertOversizedImages = true; // detect non-optimized images
     return Portal(
       child: Themed(
         child: OverlaySupport.global(
-          child: GetMaterialApp(
```
Comment (gtalha07) [inline app/lib/main.dart:70]: I think not having this wrap or replacing would not trigger the cross-signing overlay as its built around ```GetX``` components. Can you try wrapping ```MaterialApp.router``` with parent ```GetMaterial``` or maybe we have to use ```OverlaySupport``` components.

File: app/lib/models/EditGroupInfoModel.dart [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import 'package:effektio_flutter_sdk/effektio_flutter_sdk_ffi.dart';
+
+class EditGroupInfoModel {
```
Comment (gtalha07) [inline app/lib/models/EditGroupInfoModel.dart:3]: Are these models needed for ```beamer``` functionality?

**Review by gtalha07 — COMMENTED** [review]
LGTM!

File: app/lib/main.dart (hunk 1 above)
Comment (inderjeet-developer7) [reply app/lib/main.dart:70]: I am using same overlay for internet connection handling and it works fine, You can test once and if it doesn't works, we can look into it. Only a minor change would be needed as the MaterialApp.router is wrapped with OverlaySupport.global itself.

File: app/lib/models/EditGroupInfoModel.dart (hunk 2 above)
Comment (inderjeet-developer7) [reply app/lib/models/EditGroupInfoModel.dart:3]: Yes, to pass data into beamer we have to pass it in data key which takes "data as MyObject" where MyObject is a class. So these models are used there, Regardless wherever there is only one parameter to pass, I passed it directly without model class

**Pull request merged**